Automattic/wp-calypso
# Fix border hover cosmetic issues

#### Changes proposed in this Pull Request

* Fixed page selector border issues
![image](https://user-images.githubusercontent.com/3422709/161044010-187fe484-37f7-40db-b746-bdab8689b2d6.png)

![hover](https://user-images.githubusercontent.com/3422709/161045904-5ff472b1-2f08-453d-8458-5b77830bb2e3.gif)

Refactored Entire Component
![hellox](https://user-images.githubusercontent.com/3422709/161260410-54691f72-0f04-44f9-89e7-d640a5d4fd4c.gif)


#### Testing instructions
Go  through `start/do-it-for-me`
On the mage selector make sure the hover effect matches the above left design.

Related to #
